core/src/io/anuke/mindustry/ui/Links.java

```diff
@@ -38,7 +58,13 @@ private static void createLinks(){
         public LinkEntry(String name, String link, Color color){
             this.name = name;
             this.color = color;
-            this.description = Core.bundle.getNotNull("link." + name + ".description");
+            String key = "link." + name + ".description";
+            String desc = Core.bundle.getOrNull(key);
+            if(desc == null){
+                missingLocales.add(key+"\n"); // <- LINE 62
```

Comment:
Why `// <- LINE 62`?

Reply:
My bad! This was a note I added when I was troubleshooting a NullPointerException.